### DISCUSSION TITLE
[性能优化] pagespeed性能优化建议

### DISCUSSION THREAD

**Discussion by geekbb** [tangly1024/NotionNext#2080] [Ideas]
**为什么提出这个新的特性改动**
可以优化一下吗？

**描述一下你推荐的解决方案**
没有，期待大佬

**描述一下你考虑过的其它替代解决方案**
也没有

**补充说明**
https://pagespeed.web.dev/analysis/https-preview-tangly1024-com/pdqaxx5ai4?form_factor=desktop
<img width="1164" alt="CleanShot 2023-07-08 at 17 40 27@2x" src="https://github.com/tangly1024/NotionNext/assets/46196358/c23f55e2-a50f-43c6-9448-73fc67fa3b92">

**Comment by tangly1024** [comment]
感谢反馈，将持续优化此主题性能

**Comment by geekbb** [comment]
> 感谢反馈，将持续优化此主题性能
vercel自带的性能分析
<img width="1627" alt="CleanShot 2023-07-09 at 23 11 52@2x" src="https://github.com/tangly1024/NotionNext/assets/46196358/2d126d58-5080-4582-8650-e1a6fd19c865">
